tests/operator_tests.py

```diff
@@ -476,13 +467,79 @@ def test_pda_power(self):
         ex = np.power(n, [2, 3, 4])
         self.assertListEqual(p.to_list(), ex.tolist())
 
+        # Test a singleton with and without a Boolean argument
+        n = np.array([7])
+        a = ak.array(n)
+
+        p_True = ak.power(a, 3, True)
+        p_Imply = ak.power(a, 3)
+        p_False = ak.power(a, 3, False)
+        self.assertListEqual(p_True.to_list(), p_Imply.to_list())
+        self.assertListEqual(p_False.to_list(), a.to_list())
```

Comment:
since we don't use the variables again, it seems cleaner to me to put the calls in line
```python
self.assertListEqual(ak.power(a, 3, True).to_list(), ak.power(a, 3).to_list())
self.assertListEqual(ak.power(a, 3, False).to_list(), a.to_list())
```